README.md

```diff
@@ -103,17 +101,50 @@ require additional configurations applied to
 ### Prerequisites
 
 The following prerequisites are required to use the helm chart:
+- [Helm 3](https://helm.sh/docs/intro/install/) (Helm 2 is not supported) 
+- See https://github.com/kubernetes/helm for more information.
+- Splunk Enterprise 7.0 or later.
+- A minimum of one Splunk platform index ready to collect the log data. This index will be used for ingesting logs.
+- An HTTP Event Collector (HEC) token and endpoint. See the following topics for more information:
+  * https://docs.splunk.com/Documentation/Splunk/8.2.0/Data/UsetheHTTPEventCollector
+  * https://docs.splunk.com/Documentation/Splunk/8.2.0/Data/ScaleHTTPEventCollector
+- To send data to [Splunk Observability Cloud:](https://docs.splunk.com/Observability/gdi/opentelemetry/install-k8s.html)
```

Comment:
move ":" out.

Reply:
moved - it shouldn't affect the link though